### ISSUE TITLE
Story: Add Audio-Described File Link to Home & Landing Pages

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/cgov-digital-platform#4488]
### Specifications

**General**
- The addition of the audio-described file shall use a simple approach that does not change the video player itself
- The addition of the audio-described file shall assume that the video will no longer be a page-level content type
- The audio-described link paragraph will have been added to the CGDP Video Media Type prior to this ticket per #4484

**Drupal Specifications**
- The audio-described version of the video will be accessed through an external link
- The audio-described version of the video will be optional for videos

**Front-End Specifications**
- The video component shall allow users to access the audio-described version of the video
- If the video does not have an audio described version to go with it, then the designed interface to go to the audio-described version will not render

**Analytics**
- No analytics required. This component is to ensure videos are accessible. A low Click Through Rate (CTR) will not change or remove this feature.

### DoR Tasks

- [x] Content Model Drafted
- [x] Content Model Reviewed
  - [x] Developer Review
  - [x] IA Review
- [x] Content Model Approved
  - [x] Developer Approval
  - [x] IA Approval
- [x] Acceptance Criteria Drafted
- [x] Scenarios created for YAML
- [x] Acceptance Criteria Reviewed
  - [x] Developer Review
  - [x] IA Review
- [x] Acceptance Criteria Approved
  - [x] Developer Approval
  - [x] IA Approval

### DoD Tasks

- [ ] Governance Drafted
- [ ] PR Review (see PR for review and comments)
- [ ] Client review with demo

### Technical Notes

**Content Model**
- [Video Media Type](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Media-Type-Video)

**Scenarios/Comps**
- On an example Home & Landing Page for test content, use the [Cgov Homepage Comp](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50053&node-type=text&m=dev) as a reference:
  - Page URL: use the ODE URL for the Cgov homepage
  - Page Title/Components: Make it identical to the comp

### Acceptance Criteria

**CMS Experience - Adding an Embedded Video to a Home and Landing Page**
```gherkin
Scenario: A content editor adds a video to a Home and Landing Page
  Given the user is logged into the CMS as an editor
    And the user is adding a video to a Home and Landing Page
  When the user is adding a video to a Home and Landing Page
  Then the experience does not change whether the video has an audio-described link or not
```

**Front-End Experience**
```gherkin
Scenario: A user views a video with and audio-described link on a Home and Landing Page
  Given the audio-described link has been added to the video media type
    And the video with an audio-described link has been added to a Home and Landing Page
  When the video is added to a Home and Landing Page
  Then the audio-described link will appear below the video with the text, "<AD Icon> Play the audio-described version of this video."
    And it will match the comp
  When the user clicks on the link for the audio-described link
  Then they will be redirected to the external link for the audio-described video
  When the video is on a Spanish Page
  Then the audio-described link will appear below the video with the text, "<AD Icon> Escuchar la descripción del video."
  When the video does not have an audio-described link
  Then it appears the same as it would before the change from this ticket
```

### Related Tickets
* META: #4318

**Comment by bennettcc** [comment]
Moving this item out of 'on hold' - per discussion with Dev, we don't need to wait for https://github.com/NCIOCPL/cgov-digital-platform/pull/4509 to be merged before work here begins

**Comment by bennettcc** [comment]
This story should remain on hold until the below PRs are merged:

https://github.com/NCIOCPL/cgov-digital-platform/pull/4509
https://github.com/NCIOCPL/cgov-digital-platform/pull/4522

cc @karlikpj @dev-rana-publicis @andyvanavery31
